Projects/Team3/Andrii Nazaruk/HomeWork3/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace HomeWork3
+{
+    class Program
+    {
+        
+        //3. Виведіть на екран наступний текст:
+        //    To be or not to be
+        //    \ Shakespeare \
+        static void Main(string[] args)
+        {
+            
+            Console.WriteLine("To be or not to be ");
```

Comment:
Can you do this with one command - not with two?

Reply:
Just need to add \n to the end of string

Reply:
Just need to add \n to the end of a string